synapse/http/matrixfederationclient.py

```diff
@@ -519,17 +578,15 @@ def put_json(self, destination, path, args={}, data={},
             json=data,
         )
 
-        response = yield self._send_request(
-            request,
```

Comment:
nit: it's probably best to leave it as one arg on each line - it gets a bit impenetrable otherwise.